tools/pipelines/build-build-tools.yml

```diff
@@ -117,6 +117,5 @@ extends:
     taskLintName: ci:eslint
     poolBuild: Large-eastus2
     checkoutSubmodules: true
-    taskBundleAnalysis: false
```

Comment:
The parameter went away because now the "generic" `build-npm-package.yml` template doesn't to bundle analysis, only `build-npm-client-package.yml` for the build pipeline of the client release group does. No other build was leveraging this possibility of the "generic" template, and the bundle sizes for everything else we build isn't as critical, so I don't think we're losing anything important by removing the capability and simplifying that template.